assembl/static2/webpack.config.js

```diff
@@ -54,5 +84,6 @@ module.exports = {
           }
         }),
         new webpack.optimize.UglifyJsPlugin(),
+        new ExtractTextPlugin("[name].css"),
```

Comment:
A) There isn't a documentation on pattern matching of name here. Is this a webpack thing?

B) Make a comment at top of this file that CSS changes are NOT hot-module friendly now.
https://github.com/webpack-contrib/extract-text-webpack-plugin/blob/webpack-1/README.md